pkg/landscaper/execution/delete.go

```diff
@@ -90,7 +94,7 @@ func (o *Operation) checkDeletable(item executionItem, items []*executionItem) b
 		// the reconcile loop.
 		// Therefore, it should not be necessary to check again against the api server.
 		if exec.DeployItem != nil {
-			o.Log().Debug("deploy item %s depends on %s and is still present", exec.DeployItem.Name, item.Info.Name)
+			logger.WithValues("predecessor", item.Info.Name, "successor", exec.DeployItem.Name).Debug("deploy item still has a successor", exec.DeployItem.Name, item.Info.Name)
```

Comment:
Should we remove the call of WithValues and add the key/value pairs to the Debug call? 

"exec.DeployItem.Name, item.Info.Name" is a strange key/value pair. Perhaps it would be better to add this info to the message?

Reply:
`log.WithValues("foo", "bar").Info("msg")` is the same as `log.Info("msg", "foo", "bar")`, so this can be simplified.

Regarding the second point, I agree that we shouldn't use the name of the DeployItem as key, since it is not deterministic and will only mess up our opensearch filtering. For the same reason, we shouldn't put the name as part of the message - it's easier to filter for a certain log message if the message is always the same. I think we can omit the `exec.DeployItem.Name, item.Info.Name` key-value-pair, because it contains the same information which is already given via the key-value-pairs in the `WithValues` part.

Reply:
Done